cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -121,8 +147,12 @@ func (o *Options) Validate(args []string) error {
 		return errors.New("only one argument can be passed in: the name of the CertificateRequest")
 	}
 
-	if o.KeyFilename != "" && (len(o.KeyFilename) < 4 || o.KeyFilename[len(o.KeyFilename)-4:] != ".key") {
-		return errors.New("file to store private key must end in '.key'")
```

Comment:
Do we no longer want this restriction?

Reply:
We don't, the team mentioned people might prefer other file extensions, so it would be easier to just not check at all.